src/ifcconvert/ColladaSerializer.h

```diff
@@ -150,7 +150,9 @@ class ColladaSerializer : public GeometrySerializer
 	public:
 		ColladaExporter(const std::string& scene_name, const std::string& fn, ColladaSerializer *_serializer)
 			: filename(fn.c_str())
-			, stream(filename)
+            /// @todo We could use the double precision feature here, but it seems to bloat the output file quite a bit
```

Comment:
Ow, I didn't know about this `bool doublePrecision = false` argument! Well, we should probably check here if these trailing 9s can somehow be traced back to something in the IFC file, or whether it is something produced inside IfcConvert.

I traced opencollada's double writing all the way to some formatting library:
https://github.com/KhronosGroup/OpenCOLLADA/blob/f4b31b4327b0eaaf539f57c618c39be83331aebd/common/libftoa/src/Commondtoa.cpp

9s could also be an artefact of that.